codecov_auth/migrations/0037_sync_user_terms_agreement.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 4.2.2 on 2023-08-30 20:30
```

Comment:
Copy every owner objects' term agreement values (via owner profile) to its associated user's agreement columns